src/components/button.jsx

```diff
@@ -0,0 +1,7 @@
+export default function Button({ number, onClick }) {
+  return (
+    <button type="button" onClick={() => onClick(number)}>
+      {number}
+    </button>
+  );
+}
```

Comment:
```suggestion
export default function Button({ children, onClick }) {
  return (
    <button type="button" onClick={onClick}>
      {children}
    </button>
  );
}
```

이런식으로 변경해주시면 버튼 컴포넌트를 범용으로 사용이 가능하실거에요.
onClick 에 매개변수를 처리하는 로직은 상위컴포넌트에서 해주면 되겠죠?

Reply:
오 생각지도 못했는데 좋은 방법인 거 같습니다. 
이런 부분(확장성을 고려한 코드)이? 항상 궁금했었는데 좋은 피드백 감사합니다!!